src/Umbraco.Web.UI.Client/src/less/application/grid.less

```diff
@@ -118,6 +118,12 @@ body {
     }
 }
 
+@media (max-width: 500px) {
+    #search-form .form-search {
+        width: ~"calc(100% - 80px)"
```

Comment:
Somehow this doesn't compile correct to `width: calc(100% - 80px)` - it returns `width: calc(20%)`.